src/Logger-Arduino_AllPins/main.py

```diff
@@ -69,30 +72,36 @@ def __init__(self) -> None:
         self.plottype = [True for x in self.variables]  # True to plot data
         self.savetype = [True for x in self.variables]  # True to save data
 
+        # Communication Parameter
         self.port_manager = True
         self.port_types = ["COM"]
         self.port_properties = {
             "timeout": 3,
             "EOL": "\n",
             "baudrate": 115200,
         }
+        self.port_str: str = ""
+        self.driver_name: str = ""
 
+        # Device Parameter
         self.max_voltage = 5.0
-
+        self.resolution: int = 10
```

Comment:
What does a resolution of 10 mean? Is it 10 bit? In other places of the file self.resolution is rather the value range maximum, i.e. "resolution" is 1023 for 10 bit

Reply:
added a comment to clarify